README.md

```diff
@@ -255,12 +255,12 @@ This section outlines all the flags you can set to customize the data prep pipel
+| `pretrained_tokenizer` <span id="pretrained_tokenizer"></span> | str | None | Valid tokenizer key from Huggingface | The pretrained tokenizer to be used for tokenizing the input data. Loaded using transformers' [`AutoTokenizer.from_pretrained`](https://huggingface.co/docs/transformers/v4.39.3/en/model_doc/auto#transformers.AutoTokenizer.from_pretrained) method, in lieu of a `tokenizer_class`, `vocab_file` and `merges_file`. </br> </br> You also have the option of loading a tokenizer from a local file path. This includes a saved model checkpoint where a tokenizer is saved along with the model. </br> </br>**_Note_**: Llama models/tokeniers from meta are gated. You can either use a non-gated version like [this example](https://huggingface.co/NousResearch/Llama-2-7b-chat-hf), or visit the [Llama2 Model Card](https://huggingface.co/meta-llama/Llama-2-7b-hf) to request access!|
 | `tokenizer_class` | str | 'gpt2' | ['gpt2'] | Tokenizer class to use, defaults to "gpt2" (transformers.GPT2Tokenizer). If `pretrained_tokenizer` is not specified, this is required. |
 | `vocab_file` | str | None | Valid file path | The vocabulary file for the tokenizer. Should be a .json file for the tokenizer class specified by `tokenizer_class`. If `pretrained_tokenizer` is not specified, this is required. It should be a .json file for a GPT2 tokenizer. |
 | `merges_file` | str | None | Valid file path | The merges file to be used with the tokenizer class specified by `tokenizer_class`. If `pretrained_tokenizer` tokenizer is not specified, this is required. It should be a .txt file for a GPT2 tokenizer. |
 | `special_tokens_dict` | str | None | string representation of json | Any non-standard special tokens in JSON format to add to tokenizer. e.g. \"{'sep_token': \"[SEP]\"}\". Additional tokens can be also added using the "additional_special_tokens" keyword. For example, indentation encoding can be added with \"{'additional_special_tokens': [\"\t\", \"\t\t\", \"\t\t\t\"]}\". |
-| `max_seq_length` | int | 2048 | 512 for gpt2 small, 1024 for gpt-xl, 2048 for gpt3-13B. | The maximum sequence length of the model you are using. |
+| `max_seq_length` | int | 2048 | Depends on the model being trained. I.e. 512 for gpt2 small, 1024 for gpt-xl, 2048 for gpt3-13B. | The maximum sequence length of the model you are using - measured in tokens. Different models use different tokenizers which will impact the number of tokens a given sequence will be represented as. See [`pretrained_tokenizer`](#pretrained_tokenizer) above.|
```

Comment:
Should we add some information on how to find this value, as in the first table that describes this flag above?

> You can find this information in a few places, but we recommend you look at the specific model card in SambaStudio to find this value.

Reply:
Also should we mention finding this in the config of the model on HuggingFace as well ?

Reply:
Yes great call! Will add now